[2017-12-22 16:36:18. 84] [DEBUG   ] --- START computation for graph 20 5 1 (__main__:57)
[2017-12-22 16:36:18. 85] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-22 16:36:18. 85] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-22 16:36:18. 94] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-23 02:36:18. 85] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 98, in purgeDominatedStrategies
    condition3 =  m not in deleted and n not in deleted;
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-23 02:36:18. 93] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_20_ntgts/res_instance_ntgts_20_den_25_dead_5_graphix_1_ix_0.pickle (mru.iomanager:121)
[2017-12-23 02:36:18. 93] [DEBUG   ] --- END computation for graph 20 5 1 (__main__:64)
[2017-12-23 02:36:18.734] [DEBUG   ] --- START computation for graph 20 10 7 (__main__:57)
[2017-12-23 02:36:18.734] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-23 02:36:18.734] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-23 02:36:18.742] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-23 12:36:18.736] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 98, in purgeDominatedStrategies
    condition3 =  m not in deleted and n not in deleted;
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-23 12:36:18.786] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_20_ntgts/res_instance_ntgts_20_den_25_dead_10_graphix_7_ix_0.pickle (mru.iomanager:121)
[2017-12-23 12:36:18.786] [DEBUG   ] --- END computation for graph 20 10 7 (__main__:64)
[2017-12-23 12:36:19.688] [DEBUG   ] --- START computation for graph 20 19 3 (__main__:57)
[2017-12-23 12:36:19.688] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-23 12:36:19.688] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-23 12:36:19.695] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-23 22:36:19.689] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 98, in purgeDominatedStrategies
    condition3 =  m not in deleted and n not in deleted;
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-23 22:36:19.704] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_20_ntgts/res_instance_ntgts_20_den_25_dead_19_graphix_3_ix_0.pickle (mru.iomanager:121)
[2017-12-23 22:36:19.704] [DEBUG   ] --- END computation for graph 20 19 3 (__main__:64)
[2017-12-23 22:36:20.211] [DEBUG   ] --- START computation for graph 25 5 9 (__main__:57)
[2017-12-23 22:36:20.212] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-23 22:36:20.212] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-23 22:36:20.218] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-24 08:36:20.212] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 97, in purgeDominatedStrategies
    condition2 = np.array_equal(np.sort(routes[n]), np.sort(routes[m])); # we want that the routes contains the same elements
  File "/usr/lib/python2.7/dist-packages/numpy/core/fromnumeric.py", line 825, in sort
    a.sort(axis, kind, order)
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-24 08:36:20.222] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_25_ntgts/res_instance_ntgts_25_den_25_dead_5_graphix_9_ix_0.pickle (mru.iomanager:121)
[2017-12-24 08:36:20.222] [DEBUG   ] --- END computation for graph 25 5 9 (__main__:64)
[2017-12-24 08:36:20.880] [DEBUG   ] --- START computation for graph 30 5 5 (__main__:57)
[2017-12-24 08:36:20.881] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-24 08:36:20.881] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-24 08:36:20.889] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
